diff --git a/scripts/kani-std-analysis/log_parser.py b/scripts/kani-std-analysis/log_parser.py
index 32b7c7f9c1c..b9d384bc713 100755
--- a/scripts/kani-std-analysis/log_parser.py
+++ b/scripts/kani-std-analysis/log_parser.py
@@ -452,6 +452,8 @@ def parse_log_lines(
     while i < len(lines):
         if lines[i].startswith('Kani generated automatic harnesses for'):
             (i, autoharness_info) = parse_autoharness_info(lines, i)
+        else:
+            autoharness_info = {}
 
         line = lines[i].rstrip()
         i += 1
diff --git a/scripts/kani-std-analysis/std-analysis.sh b/scripts/kani-std-analysis/std-analysis.sh
index 29223388380..99d7f92628a 100755
--- a/scripts/kani-std-analysis/std-analysis.sh
+++ b/scripts/kani-std-analysis/std-analysis.sh
@@ -38,7 +38,9 @@ else
   echo
   echo "Std-Lib codegen regression only works on Linux or OSX x86 platforms, skipping..."
   echo
-  exit 0
+  TARGET="aarch64-unknown-linux-gnu"
+  WRAPPER="time"
+  # exit 0
 fi
 
 # Get Kani root
diff --git a/scripts/run-kani.sh b/scripts/run-kani.sh
index b996564989b..74afd812f4c 100755
--- a/scripts/run-kani.sh
+++ b/scripts/run-kani.sh
@@ -174,7 +174,8 @@ build_kani() {
 
 get_kani_path() {
     local build_dir="$1"
-    echo "$(realpath "$build_dir/scripts/kani")"
+    # echo "$(realpath "$build_dir/scripts/kani")"
+    echo "kani"
 }
 
 # Run kani list with JSON format and process with jq to extract harness names and total number of harnesses.
@@ -261,7 +262,7 @@ main() {
         setup_kani_repo "$REPO_URL" "$build_dir" "$BRANCH_NAME" "$commit"
 
         # Build project
-        build_kani "$build_dir"
+        # build_kani "$build_dir"
 
         echo "Kani build completed successfully!"
     fi
@@ -322,11 +323,11 @@ main() {
     elif [[ "$run_command" == "metrics" ]]; then
         local current_dir=$(pwd)
         echo "Running Kani list command..."
-        if [[ "$with_autoharness" == "true" ]]; then
-            "$kani_path" autoharness -Z autoharness --list $unstable_args --std ./library --format json
-        else
-            "$kani_path" list $unstable_args ./library --std --format json
-        fi
+        # if [[ "$with_autoharness" == "true" ]]; then
+        #     "$kani_path" autoharness -Z autoharness --list $unstable_args --std ./library --format json
+        # else
+        #     "$kani_path" list $unstable_args ./library --std --format json
+        # fi
         pushd scripts/kani-std-analysis
         echo "Running Kani's std-analysis command..."
         ./std-analysis.sh $build_dir
